src/main/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackService.java

```diff
@@ -71,23 +61,18 @@ public CallbackService(final SecurityUtils securityUtils,
         }
 
         final CallbackRequest callbackRequest = new CallbackRequest(caseDetails,
-                                                                    caseDetailsBefore,
-                                                                    caseEvent.getId(),
-                                                                    ignoreWarning);
-        final List<Integer> retries = CollectionUtils.isEmpty(callbackRetries) ? defaultRetries : callbackRetries;
-
-        for (Integer timeout : retries) {
-            final Optional<ResponseEntity<CallbackResponse>> responseEntity = sendRequest(url,
-                                                                                          CallbackResponse.class,
-                                                                                          callbackRequest,
-                                                                                          timeout);
-            if (responseEntity.isPresent()) {
-                return Optional.of(responseEntity.get().getBody());
-            }
+            caseDetailsBefore,
+            caseEvent.getId(),
+            ignoreWarning);
+        final Optional<ResponseEntity<CallbackResponse>> responseEntity = sendRequest(url, CallbackResponse.class, callbackRequest);
+        if (responseEntity.isPresent()) {
+            return Optional.of(responseEntity.get().getBody());
```

Comment:
consider replacing this logic with:
```        responseEntity.map(re -> responseEntity.get().getBody()).orElseThrow(() -> new CallbackException("Unsuccessful callback to " + url));
```

Reply:
sorry the above does not compile, this one does:
```
return responseEntity.map(re -> Optional.ofNullable(re.getBody())).orElseThrow(() -> new CallbackException("Unsuccessful callback to " + url));
```

Reply:
applied with one correction `return responseEntity.map(re -> Optional.of(re.getBody())).orElseThrow(() -> new CallbackException("Unsuccessful callback to " + url));`